src/cljc/jinteki/validator.cljc

```diff
@@ -280,3 +280,13 @@
     (if status
       status
       (calculate-deck-status deck)))
+
+(defn legal-deck?
+ ([deck] (legal-deck? deck (:format deck)))
+ ([deck fmt]
+   (get-in deck
```

Comment:
Instead of nesting `get-in`s, let's break each of these out into a `let` block for readability.

Reply:
Looks like you took this out of the gamelobby file, which is great, so thanks for that. Still good to reformat when moving code like this. Better to get it clean while we're here!